src/test/java/org/springframework/data/couchbase/core/mapping/MappingCouchbaseConverterTests.java

```diff
@@ -1184,10 +1184,12 @@ class Entity {
 			@IdSuffix(order = 1) public String suffix = "222";
 		}
 		Entity entity = new Entity();
+		entity.id = entity.prefix + '.' + entity.prefix1 + '.' + entity.someId + '.' + entity.suffix + '.' + entity.suffix1;
```

Comment:
The test is for the construction of converted.getId() by the converter.  When entity.id is already it us used instead of constructing it.